contents/blog/clickhouse-announcement.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2021-08-13
+title: Why we’re moving you to ClickHouse - Benefits and tradeoffs
```

Comment:
We need a better title, especially if you don't know what Clickhouse is. e.g. 'We just made PostHog Open Source 1000x more scalable'

Reply:
Amended